app/models/user.rb

```diff
@@ -0,0 +1,15 @@
+class User < ApplicationRecord
+  before_save :downcase_email
+
+  validates :name, presence: true
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
+  validates :email, presence: true, length: {maximum: 255}, format: {with: VALID_EMAIL_REGEX}
```

Comment:
```suggestion
  validates :email, presence: true, length: {maximum: 255}, format: {with: VALID_EMAIL_REGEX}, uniqueness: true
```
Why you have to make 2 validate email, when you need one

Reply:
ar, just know it. fixed